js/src/views/Web/web.js

```diff
@@ -0,0 +1,209 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+
+import styles from './web.css';
+
+export default class Web extends Component {
+  static contextTypes = {
+    api: PropTypes.object.isRequired
+  }
+
+  state = {
+    token: null,
+    isLoading: true,
+    displayedUrl: this.lastAddress(),
+    url: this.lastAddress()
+  };
+
+  handleUpdateUrl = (url) => {
+    try {
+      window.localStorage.setItem('parity-web-lastAddress', url);
+    } catch (e) {}
+
+    this.setState({
+      isLoading: true,
+      displayedUrl: url,
+      url: url
+    });
+  };
+
+  handleOnRefresh = (ev) => {
+    // TODO [ToDr]
+    this.setState({
+      isLoading: true,
+      url: `${this.state.displayedUrl}?t=${Date.now()}`
+    });
+  };
+
+  handleIframeLoad = (ev) => {
+    this.setState({
+      isLoading: false
+    });
+  };
+
+  lastAddress () {
+    let res = null;
+    try {
+      res = window.localStorage.getItem('parity-web-lastAddress');
```

Comment:
As per above, would prefer the key a constant and using `store` (just a bit less boilerplate)